configs/keycloak_bridge.yml

```diff
@@ -192,4 +196,4 @@ register-techuser-client-id: admin-cli
 register-enduser-client-id: selfserviceid
 recaptcha-url: https://www.google.com/recaptcha/api/siteverify
 recaptcha-secret: theverymysterioussecretfortherecaptchaverifyoperation
-sse-public-url: https://sse.trustid.ch
+sse-public-url: https://sse.trustid.ch
```

Comment:
why do we remove the new line ? Usually it is prefered to have one no ?

Reply:
DONE